qt/applications/workbench/workbench/plugins/editor.py

```diff
@@ -76,11 +79,14 @@ def app_closing(self):
     def get_plugin_title(self):
         return "Editor"
 
-    def readSettings(self, _):
-        pass
+    def readSettings(self, settings):
+        try:
+            self.editors.prev_session_tabs = settings.get('Editor/SessionTabs')
```

Comment:
Can we define the settings key once as a class-level variable like `SETTINGS_KEY = 'Editors/SessionsTabs'` to save repeating ourselves?